en/docs/develop/product-apis/gateway-apis/gateway-v1/guide.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started
+
+WSO2 API Manager is a complete solution for publishing APIs, creating and managing a developer community, and for scalably routing API traffic. It leverages proven, production-ready, integration, security and governance components from WSO2 Enterprise Service Bus, WSO2 Identity Server, and WSO2 Governance Registry. Moreover, it is powered by WSO2 Business Activity Monitor, thereby making WSO2 API Manager ready for any large-scale deployments right away.
+
+The REST API is implemented based on REST best practices and specifications as a CXF REST web application running on WSO2 API Manager. API development is started with a Swagger specification with a contract-first approach. Please see the full [Swagger definition](https://raw.githubusercontent.com/wso2/carbon-apimgt/v6.6.163/components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/resources/admin-api.yaml), which is written using Swagger 2.0. This can be also retrieved from the Web app itself using the URL `https://<host-name[:port]>/api/am/admin/v0.16/swagger.json`.
+
+The API comes with a pluggable security mechanism. Since API security is implemented as a CXF handler, if you need to plug a custom security mechanism, you can write your own handler and add it to the web service.
+
+Before invoking the API with the access token, obtain the consumer key/secret key pair by calling the dynamic client registration endpoint. You can request an access token with the preferred grant type. An example is shown below,
+
+```
+curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d @payload.json https://localhost:9443/client-registration/v0.16/register
+```
+
+Sample request:
+
+```
+{
+   "callbackUrl":"www.google.lk",
+   "clientName":"rest_api_admin",
+   "owner":"admin",
+   "grantType":"password refresh_token",
+   "saasApp":true
+}
+```
+
+Sample response:
+
+```
+{
+   "callBackURL":"www.google.lk",
+   "jsonString":"{ \"username\":\"admin\", \"redirect_uris\":\"www.google.lk\", \"client_name\":\"admin_rest_api_admin\", \"grant_types\":\"authorization_code password refresh_token iwa:ntlm urn:ietf:params:oauth:grant-type:saml2-bearer client_credentialsimplicit\" }",
+   "clientName":null,
+   "clientId":"HfEl1jJPdg5tbtrxhAwybN05QGoa",
+   "clientSecret":"l6c0aoLcWR3fwezHhc7XoGOht5Aa"
+}
+```
+
+During the API invocation process request, click the CXF handler first, which calls an introspection API to validate the token. Generate the access token using the already created OAuth application. A sample call to generate the access token is shown below.
```

Comment:
When an API is invoked, the handler invokes an introspection API to validate the token.  

The flow of events are not in order. Can you please revsit it?